packages/amplify-e2e-core/src/categories/api.ts

```diff
@@ -528,7 +564,7 @@ export function addRestApi(cwd: string, settings: any) {
 
 const allAuthTypes = ['API key', 'Amazon Cognito User Pool', 'IAM', 'OpenID Connect'];
 
-export function addApi(projectDir: string, settings?: any) {
+export function addApi(projectDir: string, settings?: any, transformerVersion: number = 2) {
```

Comment:
This particular function makes the new implementation a bit awkward. You could add support for `transformerVersion` to this `settings` object and then delete the property from the object so that the external API is more consistent but the internal functionality doesn't need to change.

Reply:
updated